build-support/bin/travis-ci.sh

```diff
@@ -1,58 +0,0 @@
-#!/usr/bin/env bash
-# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
-# Licensed under the Apache License, Version 2.0 (see LICENSE).
-
-# TravisCI-specific environment fixups can live in this script which forwards to the generic ci
-# script post fixups.
-
-set -euo pipefail
-
-cd "$(git rev-parse --show-toplevel)"
-
-export RUNNING_VIA_TRAVIS_CI_SCRIPT=1
```

Comment:
This env var is used in `build-support/githooks/pre-commit`:

https://github.com/pantsbuild/pants/blob/a3cb9c08f86c193f76336284580ff302981388d7/build-support/githooks/pre-commit#L70-L73

I think we'll want to add this to the global env entries for `.travis.yml`.

Reply:
I've switched to using the `TRAVIS` env var instead: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables